app/app.go

```diff
@@ -137,6 +145,7 @@ var (
 		stakingtypes.NotBondedPoolName: {authtypes.Burner, authtypes.Staking},
 		govtypes.ModuleName:            {authtypes.Burner},
 		ibctransfertypes.ModuleName:    {authtypes.Minter, authtypes.Burner},
+		claimtypes.ModuleName:          nil,
```

Comment:
@aleem1314 dont we need the minting permission here? Did you discuss with Sai?

Reply:
Okay i will add permissions back